pkg/install/templates/flux-deployment.yaml.tmpl

```diff
@@ -159,6 +159,9 @@ spec:
 
         # Tell flux it has readonly access to the repo (default `false`)
         # - --git-readonly
+{{ if .GitReadOnly }}
```

Comment:
For neatness in the output, it would be nice to output _either_ the arg or it's commented-out analogue. E.g.,

```
{{ if .GitReadOnly }}
  - --git-readonly
{{ else }}
  # - --git-readonly
{{ end }}
```